app/src/main/java/de/danoeh/antennapod/adapter/SubscriptionsRecyclerAdapter.java

```diff
@@ -52,12 +52,18 @@ public class SubscriptionsRecyclerAdapter extends SelectableAdapter<Subscription
     private List<NavDrawerData.DrawerItem> listItems;
     private NavDrawerData.DrawerItem selectedItem = null;
     int longPressedPosition = 0; // used to init actionMode
+    private final SharedPreferences prefs;
+
+    private static final String PREF_NUM_COLUMNS = "columns";
+    private final int defaultColumnCount;
 
     public SubscriptionsRecyclerAdapter(MainActivity mainActivity) {
         super(mainActivity);
         this.mainActivityRef = new WeakReference<>(mainActivity);
         this.listItems = new ArrayList<>();
         setHasStableIds(true);
+        this.prefs = mainActivity.getSharedPreferences(PREF_NUM_COLUMNS, Context.MODE_PRIVATE);
```

Comment:
This does not load the same preferences that the Fragment saves. Please store just an int here in the adapter and pass the value from within the fragment.

Reply:
Value is passed from fragment -> adapter now